### ISSUE TITLE
readme.txt instead of .md

### ISSUE THREAD

**Issue by bueltge** [inpsyde/boilerplate#2]
Hi David.
Maybe I'm wrong. but we should create in the w.org assets directory a readme in txt-format, not md.
from https://github.com/inpsyde/boilerplate/blob/master/_templates/w-org-assets/README.md.template

**Comment by dnaber-de** [comment]
I agree that we need a `readme.txt` in case of type theme or plugin. Yet not sure where to place it, that depends on a possible automation of a deployment to the w.org SVN repos.
